communication/data_logging.hpp

```diff
@@ -112,8 +104,10 @@ typedef struct
 	
 	uint32_t sys_id;							///< the system ID
 	
+	Console<File>* console;						///< The pointer to the console containing the file to write data to
```

Comment:
I did it that way to avoid creating a class for data_logging. 
My goal was to ease at most the creation of multiple entities of data_logging (therefore trying to avoid to have to pass file object and change the constructor of boards.